src/main/resources/application.yaml

```diff
@@ -74,6 +74,7 @@ springdoc:
 replication:
   username: ${REPLICATION_USERNAME:repl_user}
   password: ${REPLICATION_PASSWORD:repl_password}
+  subscriptionUrl: postgresql://${POSTGRES_CLUSTER_HOST:192.168.49.2}:${POSTGRES_PORT:5432}
```

Comment:
It's not essential but would be good to have the host as `localhost`

Reply:
changed the default now back to localhost.